pkg/cloudprovider/aws/ami.go

```diff
@@ -83,16 +84,42 @@ func (p *AMIProvider) getAMIID(ctx context.Context, query string) (string, error
 	return ami, nil
 }
 
-func (p *AMIProvider) getSSMQuery(instanceType cloudprovider.InstanceType, version string) string {
-	var amiSuffix string
+// getAL2Alias returns a properly-formatted alias for an Amazon Linux AMI from SSM
+func (p *AMIProvider) getAL2Alias(version string, instanceType cloudprovider.InstanceType) string {
+	amiSuffix := ""
 	if !instanceType.NvidiaGPUs().IsZero() || !instanceType.AWSNeurons().IsZero() {
 		amiSuffix = "-gpu"
 	} else if instanceType.Architecture() == v1alpha5.ArchitectureArm64 {
-		amiSuffix = "-arm64"
+		amiSuffix = fmt.Sprintf("-%s", instanceType.Architecture())
 	}
 	return fmt.Sprintf("/aws/service/eks/optimized-ami/%s/amazon-linux-2%s/recommended/image_id", version, amiSuffix)
 }
 
+// getBottlerocketAlias returns a properly-formatted alias for a Bottlerocket AMI from SSM
+func (p *AMIProvider) getBottlerocketAlias(version string, instanceType cloudprovider.InstanceType) string {
+	arch := "x86_64"
+	if instanceType.Architecture() == v1alpha5.ArchitectureArm64 {
+		arch = instanceType.Architecture()
+	}
+	return fmt.Sprintf("/aws/service/bottlerocket/aws-k8s-%s/%s/latest/image_id", version, arch)
+}
+
+func (p *AMIProvider) getSSMQuery(ctx context.Context, constraints *v1alpha1.Constraints, instanceType cloudprovider.InstanceType, version string) string {
+	ssmQuery := p.getAL2Alias(version, instanceType)
```

Comment:
Could you add documentation changes as part of this PR? We should call out that you will be defaulted to the EKS-optimized AMI.

Reply:
This is optional - we can have this be a separate PR too. Just want to make sure we include it before we do another release :)